example/src/main/java/org/apache/rocketmq/example/benchmark/BatchProducer.java

```diff
@@ -360,17 +364,17 @@ public LongAdder getSendMessageFailedCount() {
 
     public void start() {
 
-        timer.scheduleAtFixedRate(new TimerTask() {
+        executorService.scheduleAtFixedRate(new TimerTask() {
```

Comment:
The code can be more concise, such as using a lambda expression

Reply:
Good idea